packages/kolibri-tools/lib/i18n/intl_code_gen.js

```diff
@@ -123,11 +119,7 @@ const intlFooter = `
 const intlModule =
   commonHeader + intlHeader + languageInfo.map(generateIntlItems).join('') + intlFooter;
 
-const intlModulePath = path.resolve(
-  __dirname,
-  '../../kolibri/core/assets/src/utils/intl-locale-data.js'
-);
-
+const intlModulePath = path.resolve('./intl-locale-data.js');
```

Comment:
This seems to have resulted in the intl-locale-data file being put in the root of the repo - this seems like the wrong place.

Reply:
yes, `intl-locale-data.js` and `vue-intl-locale-data.js` show as new rather than updated files